src/install/f5/whatsNext.mdx

```diff
@@ -0,0 +1,2055 @@
+          HOSTNAME: {{F5_HOST}}
+          PORT: 443
+          USERNAME: f5_user
+          PASSWORD: f5_password
+        interval: 15s
+        labels:
+          env: production
+          role: load_balancer
+    ```
+  </Collapser>
+
+  <Collapser
+    id="filters"
+    title="Metrics-only with partition filtering"
+  >
+    This configuration only collects metrics and adds `"MyOtherPartition"` to the list of partitions to be sampled. By default, the integration only samples the `"Common"` partition:
+
+    ```
+    integrations:
+      - name: nri-f5
+        env:
+          METRICS: "true"
+          HOSTNAME: {{F5_HOST}}
+          PORT: 443
+          USERNAME: f5_user
+          PASSWORD: f5_password
+          PARTITION_FILTER:  '["Common","MyOtherPartition"]'
+        interval: 15s
+        labels:
+          env: production
+          role: load_balancer
+    ```
+  </Collapser>
+
+  <Collapser
+    id="multi-instance"
+    title="Multi-instance monitoring"
+  >
+    This configuration monitors multiple F5 servers from the same integration. The first instance (`HOSTNAME: 1st_f5_host`) collects metrics and inventory, while the second instance (`HOSTNAME: 2nd_f5_host`) only collects metrics.
+
+    ```
+    integrations:
+      - name: nri-f5
+        env:
+          METRICS: "true"
+          HOSTNAME: 1st_f5_host
+          PORT: 443
+          USERNAME: f5_user
+          PASSWORD: f5_password
+        interval: 15s
+        labels:
+          env: production
+          role: load_balancer
+      - name: nri-f5
+        env:
+          INVENTORY: "true"
+          HOSTNAME: 1st_f5_host
+          PORT: 443
+          USERNAME: f5_user
+          PASSWORD: f5_password
+        interval: 60s
+        labels:
+          env: production
+          role: load_balancer
+        inventory_source: config/f5
+
+      - name: nri-f5
+        env:
+          METRICS: "true"
+          HOSTNAME: 2nd_f5_host
+          PORT: 443
+          USERNAME: f5_user
+          PASSWORD: f5_password
+        interval: 15s
+        labels:
+          env: production
+          role: load_balancer
+    ```
+  </Collapser>
+</CollapserGroup>
+
+## Find and use data [#find-and-use]
+
+To find your integration data, go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Infrastructure > Third-party services** and select one of the F5 BIG-IP integration links.
+
+F5 BIG-IP data is attached to the following event types:
+
+* `F5BigIpSystemSample`
+* `F5BigIpVirtualServerSample`
+* `F5BigIpPoolSample`
+* `F5BigIpPoolMemberSample`
+* `F5BigIpNodeSample`
+
+For more on how to find and use your data, see [Understand integration data](/docs/infrastructure/integrations/find-use-infrastructure-integration-data).
+
+## Metric data [#metrics]
+
+The F5 BIG-IP integration collects the following metric data attributes. Some metric name are prefixed with a category indicator and a period, such as `system.`, `virtualserver.`, or `pool.`.
```

Comment:
Why do you put inventory data in collapsers, but not the tables for metric data? Is it worth paralleling the sections?